tests/functional/adapter/test_backup_table.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+from dbt.tests.util import run_dbt
+
+
+_MODELS = {
```

Comment:
I really think -- as annoying as this is -- we  should have these be split into individual model variables and then included where necessary. Right now, it's hard to know what's needed where and the readability is lower. What happens if any of these extended the line past 100 chars for example? This may be needed on a new model. I'd rather we follow the existing convention.

Also myself wouldn't really care if you extracted the model definitions into their own variables and built `_MODELS` like you have here.

Reply:
I didn't realize a module variable per model was the convention. I can make that change. I went with the more compact version simply because it fit, as you noted. That being said, I think they're all needed for the class, so I would just move this into `def models(self)` referencing the variable instead of the string literal. Also, running it at the class level makes it that I only need to run dbt once, which is the long pole in the tent.